apps/api/views/homework.py

```diff
@@ -49,6 +49,16 @@ def create(self, request, *args, **kwargs):
     def perform_create(self, serializer):
         serializer.is_valid(raise_exception=True)
         new_sub = serializer.save()
+
+        # Move grade and other answers from most recent submission to this new submission
```

Comment:
should we maybe be copying this instead of pointing it to latest? I think pointing to latest is the best solution, probably.

anything weird happen if the latest homework is deleted? will the grade go back to the previous homework?

Reply:
It would be a bit more robust if we copied it. This is still not an ideal solution. Just to get something working for Adrien. I was hoping to eventually implement a system that didn't tie grades to single submissions but rather between students and homework definitions. This way the grade would not have to be moved or copied. It would stay static, and would not be dependent on submissions even existing, necessarily.

If you'd like me to change it, so that the grade is copied in the meantime, I'd be happy to for the sake of robustness.